tests/framework/scheduler.py

```diff
@@ -115,7 +115,8 @@ def pytest_runtestloop(self, session):
                     "/functional/test_initrd.py",
                     "/functional/test_max_vcpus.py",
                     "/functional/test_rate_limiter.py",
-                    "/functional/test_signals.py"
+                    "/functional/test_signals.py",
+                    "/build/test_coverage.py"
```

Comment:
Wouldn't changing the directory of the coverage files fix this problem? I was thinking this way we don't need to compromise on the total build time.

Reply:
I think we need to first make the changes to allow unit-tests in general to be run in parallel.

For example, there are some api-server unittests that use some hardcoded absolute paths, so changing the relative directory for the coverage isn't going to be enough.

Reply:
Hmm, okay. We should probably track this issue somewhere.